app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/ChinesePinyinKeyboard.java

```diff
@@ -49,7 +49,7 @@ public CandidatesResult getCandidates(String aComposingText) {
             return null;
         }
 
-        // Autocomplete when space is clicked
+        // Autocomplete when special characters are clicked
         boolean autocomponse = aComposingText.endsWith(" ")
                                || aComposingText.endsWith("，")
                                || aComposingText.endsWith("。");
```

Comment:
It seems like `!`, `?`, and long sound symbol also need to be added to this rule.